### ISSUE TITLE
Outdated .skill file format (minor)

### ISSUE THREAD

**Issue by PreConceptor** [RkShaRkz/Starsector-IntelliJ-Template#13]
ver 3.3.6

Issue: field_repairs.skill file uses an outdated format so it doesn't unlock the Collapsed Cargo Holds hullmod when unlocking the Field Repairs skill.

Solution: I made a version with updated formatting (attached as .txt, git won't allow the .skill file). Replacing the old one makes it function again.

[field_repairs.txt](https://github.com/user-attachments/files/15979650/field_repairs.txt)

**Comment by RkShaRkz** [comment]
hm, alright, but where is the "field repairs" mentioned in this new one?

new:
```
{
	"effectGroups":[
		{
			"requiredSkillLevel":1,
			"effectBasedOnLevel":false,
			"effects":[
				{"type":"HULLMOD_UNLOCK", "hullmods":{"vayra_collapsedcargo":1}},
			]
		},
	],
}
```

old:
```
{
	"id":"field_repairs",
	"effectGroups":[
		{
			"name":"Level 2",
			"effects":[
				{"type":"HULLMOD_UNLOCK", "hullmods":{"vayra_collapsedcargo":2}},
			]
		},
	],
}
```

**Comment by PreConceptor** [comment]
The file name. It doesn't need to be mentioned in the file.

Here's the Officer Management skill file from Nexerelin, it adds the bonus operative capacity effect. 

[officer_management.txt](https://github.com/user-attachments/files/16044601/officer_management.txt)

Also no id within the file but it works. It's the format every current mod I've seen uses to add things to existing skills. I've been using the fixed one myself for more than a year. The old format predates the update that changed skills and introduced elite skills, so it looks like the unlock is present in-game but its listed under 'Level 2' in the skill window. Since Level 2 skills in the old sense no longer exist it doesn't unlock.

**Comment by RkShaRkz** [comment]
I reproed the issue, and the hullmod is unavailable for me as well.
My idea was to try this and see whether that works 
```
{
	"id":"field_repairs",
	"effectGroups":[
		{
			"requiredSkillLevel":1,
			"effectBasedOnLevel":false,
			"effects":[
				{"type":"HULLMOD_UNLOCK", "hullmods":{"vayra_collapsedcargo":1}},
			]
		},
	],
}
```

whether the problem was in the `2` or in the 'level 2' - we'll end up with a working solution.

**Comment by RkShaRkz** [comment]
Can confirm this fixes it. Resolved in commit  183080c

**Comment by RkShaRkz** [comment]
Oh right @PreConceptor - my idea didn't work, we went with yours. thanks :)
